experimental/cloudstrapper/playbooks/roles/vars/mantle.yaml

```diff
@@ -0,0 +1,12 @@
+
+#------------------
+#Source-specific Variables - Used for Devops
+#------------------
+
+mantle_git_repo: git@github.com:fbcinternal/mantle.git
```

Comment:
is this needed or can this be cleaned up ?

Reply:
Cleaned up.